### DISCUSSION TITLE
Un badge compagnon

### DISCUSSION THREAD

**Discussion by Easygamer01s** [Crownicles/Crownicles#1666] [Nouvelles idées]
Pour un certain montant, il est possible, à une seule reprise, d'acheter un badge assigné à un autre joueur.

**Comment by Akutana** [comment]
Ça enlève tout l'intérêt des badges :pQuack:

**Reply by Easygamer01s** [reply]
Il s'agirait d'un badge à part comme le moneymouth :)

**Reply by Akutana** [reply]
Quels badges par exemple ?

**Reply by Easygamer01s** [reply]
Un nouveau badge, pas d'anciens badge, que j'achète pour que mon ami puisse l'obtenir sur son profil :)

**Reply by Akutana** [reply]
Jcomprends pas comment ça marche du coup

**Comment by BastLast** [comment]
Ptdr et il coûterait genre 100k.

**Reply by BastLast** [reply]
Money mounth mais en plus cher et achetable que mar les autres

**Reply by voltou91** [reply]
Moi je peux l'acheter 3 fois, mouhahahah
